projects/plugins/jetpack/extensions/store/membership-products/reducer.js

```diff
@@ -23,6 +30,34 @@ export default function reducer( state = DEFAULT_STATE, action ) {
 				...state,
 				connectedAccountDefaultCurrency: action.connectedAccountDefaultCurrency,
 			};
+		case 'SET_SOCIAL_FOLLOWER_COUNT':
+			return {
+				...state,
+				counts: {
+					...state.counts,
+					socialFollowers: action.socialFollowers,
+				},
+			};
+		case 'SET_EMAIL_SUBSCRIBER_COUNT':
+			return {
+				...state,
+				counts: {
+					...state.counts,
+					emailSubscribers: action.socialFollowers,
```

Comment:
might be an error there

```suggestion
					emailSubscribers: action.emailSubscribers,
```